src/System.Net.Primitives/tests/UnitTests/CookieInternalTest.cs

```diff
@@ -68,5 +68,94 @@ public void CookieParserGetServer_SetCookieHeaderValue_Success(string cookieStri
             int expectedCookieCount = expectedStrings.Length >> 1;
             Assert.Equal(expectedCookieCount, cookieCount);
         }
+
```

Comment:
Why add the new tests to Unit Test project? `CookieContainer` is a public class which can be directly initialized (unlike the above test testing internal `CookieParser`), and we are testing it's behavior here.

Maybe it's more appropriate to put new tests under `corefx/src/System.Net.Primitives/tests/FunctionalTests`